zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/validator/GroupRequestValidator.java

```diff
@@ -19,20 +19,24 @@ public final class GroupRequestValidator {
 
   private GroupRequestValidator() {}
 
-  public static Optional<ProblemDetail> validateGroupName(final String name) {
+  public static Optional<ProblemDetail> validateUpdateRequest(final GroupUpdateRequest request) {
     return validate(
         violations -> {
-          if (name == null || name.isBlank()) {
+          if (request.getChangeset().getName() == null || request.getChangeset().getName().isBlank()) {
             violations.add(ERROR_MESSAGE_EMPTY_ATTRIBUTE.formatted("name"));
           }
+          if (request.getChangeset().getDescription() == null) {
+            violations.add(ERROR_MESSAGE_EMPTY_ATTRIBUTE.formatted("description"));
+          }
```

Comment:
❓ Is this true? If a user doesn't provide a description shouldn't that be fine? We can default to an empty string?

Reply:
I followed the same validation of the TenantUpdateRequest

Reply:
I thought the same 👍